projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1276,62 +1428,71 @@ static function revisions_meta_box( $safecss_post ) {
 
 		$posts_per_page = $show_all_revisions ? $max_revisions : 6;
 
-		$revisions = new WP_Query( array(
-			'posts_per_page' => $posts_per_page,
-			'post_type' => 'revision',
-			'post_status' => 'inherit',
-			'post_parent' => $safecss_post['ID'],
-			'orderby' => 'date',
-			'order' => 'DESC'
-		) );
+		$revisions = new WP_Query(
+			array(
+				'posts_per_page' => $posts_per_page,
+				'post_type'      => 'revision',
+				'post_status'    => 'inherit',
+				'post_parent'    => $safecss_post['ID'],
+				'orderby'        => 'date',
+				'order'          => 'DESC',
+			)
+		);
 
-		if ( $revisions->have_posts() ) { ?>
-			<ul class="post-revisions"><?php
+		if ( $revisions->have_posts() ) {
+			?>
+			<ul class="post-revisions">
+			<?php
 
 			global $post;
 
 			while ( $revisions->have_posts() ) :
 				$revisions->the_post();
 
-				?><li>
+				?>
+				<li>
 					<?php
-						echo wp_post_revision_title( $post );
+						echo wp_post_revision_title( $post ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 
-						if ( ! empty( $post->post_excerpt ) )
-							echo ' (' . esc_html( $post->post_excerpt ) . ')';
+					if ( ! empty( $post->post_excerpt ) ) {
+						echo ' (' . esc_html( $post->post_excerpt ) . ')';
+					}
 					?>
-				</li><?php
+				</li>
+				<?php
 
 			endwhile;
 
-			?></ul><?php
+			?>
+			</ul>
+			<?php
 
-			if ( $revisions->found_posts > 6 && !$show_all_revisions ) {
+			if ( $revisions->found_posts > 6 && ! $show_all_revisions ) {
 				?>
 				<br>
-				<a href="<?php echo add_query_arg( 'show_all_rev', 'true', menu_page_url( 'editcss', false ) ); ?>"><?php esc_html_e( 'Show all', 'jetpack' ); ?></a>
+				<a href="<?php echo add_query_arg( 'show_all_rev', 'true', menu_page_url( 'editcss', false ) ); ?>"><?php esc_html_e( 'Show all', 'jetpack' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></a>
```

Comment:
The ignore comment should go in the other `<?php  ?>`. But even better, just add the escaping.
```suggestion
				<a href="<?php echo esc_url( add_query_arg( 'show_all_rev', 'true', menu_page_url( 'editcss', false ) ) ); ?>"><?php esc_html_e( 'Show all', 'jetpack' ); ?></a>
```